### PR TITLE
add react_facebook_login

### PR DESCRIPTION
**Pull request by maheshkurma**
added - react Facebook login

### CODE REVIEW

**Review by sarath-yerra — DISMISSED** [review]

**Comment by sarath-yerra** [comment]
We need to merge this after merging back end PR (https://github.com/midohiofoodbank/freshtrak-registration-api/pull/39)

**Review by davidpickering — APPROVED** [review]
Adds Facebook Login to the site and integrates with changes to the registration API ([add service to verify facebook auth](https://github.com/midohiofoodbank/freshtrak-registration-api/pull/39)).

**Pull request merged**
